src/durabletask/azext_durabletask/custom.py

```diff
@@ -1,14 +1,45 @@
 # --------------------------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
-#
-# Code generated by aaz-dev-tools
 # --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
 
-# pylint: disable=too-many-lines
-# pylint: disable=too-many-statements
+import httpx
+from azure.cli.core.commands.client_factory import get_subscription_id
+from .aaz.latest.durabletask.scheduler import Show
 
-from knack.log import get_logger
 
+def list_orchestrations(cmd, resource_group_name, scheduler_name, taskhub_name, max_items=100):
+    # Get FQDN of the scheduler
 
-logger = get_logger(__name__)
+    scheduler = Show(cli_ctx=cmd.cli_ctx)(command_args={
+        "resource_group": resource_group_name,
+        "name": scheduler_name,
+        "subscription": get_subscription_id(cmd.cli_ctx)
+    })
+
+    endpoint = scheduler['properties']['endpoint']
+    endpoint += "/v1/taskhubs/orchestrations/query"
+    grabbed_token = f"Bearer {get_bearer_token(cmd.cli_ctx)}"
+
+    headers = {
+        'Content-Type': 'application/json',
+        'Authorization': grabbed_token,
+        'x-taskhub': taskhub_name
+    }
+
+    payload = {"filter": {}, "pagination": {"startIndex": 0, "count": max_items}, "sort": [{"column": "LAST_UPDATED_AT", "direction": "DESCENDING_SORT"}]}
```

Comment:
The start index should be made an optional argument, too, else callers could never get the "next" page of orchestrations.  (Else, they'd have to make the call over and over, specifying larger and larger counts, which seems inefficient.)